src/source/location.d

```diff
@@ -46,17 +46,17 @@ public:
 	}
 
 	auto spanTo(Location end) in {
-		import std.conv;
+		import std.format;
 		assert(stop.offset <= end.stop.offset,
-		       to!string(stop.offset) ~ " > " ~ to!string(end.stop.offset));
+		       format("%u > %u", stop.offset, end.stop.offset));
```

Comment:
Most of the codebase uses `format!"pattern"` so let's stick with this.

Why %u instead of %d ?

Reply:
`.offset` is unsigned.
Docs say `%d` … “To be formatted as a signed decimal integer.”

Reply:
> so let's stick with this

done

Reply:
> signed decimal integer

what’s your take on this? Am I misinterpreting the documentation?

Reply:
I think just using %s is simpler. D, thanks to templates, is able to figure it out.

The "problem" with %u is what happens if/when we change the type of the argument?

Reply:
> I think just using %s is simpler.

alright, implemented :)